cli/start.go

```diff
@@ -17,7 +19,22 @@ var (
 )
 
 func startRun(cmd *cobra.Command, args []string) {
-	req, err := createRequest(args)
+	group := ""
+	switch len(args) {
+	case 1:
+		group = args[0]
+	case 0:
+		fallthrough
+	default:
+		cmd.Help()
+		os.Exit(1)
+	}
+
+	newRequestConfig := controller.DefaultRequestConfig()
```

Comment:
3 operations against the controller to do the `start` - feels a bit overly complex. I would suggest to rethink this any maybe we can come up with sthg better (outside of this PR ofc)